_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -34161,42 +34172,48 @@
 	pixel_y = 22
 	},
 /turf/open/floor/plasteel,
-/area/science/mineral_storeroom)
+/area/science/mixing)
 "bFu" = (
-/obj/structure/ore_box,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,
+/obj/machinery/power/apc{
```

Comment:
Are there two APCs in this area now?

Reply:
Oops. I removed the duplicate APC and moved the air alarm to that tile.